linkerd/app/src/identity.rs

```diff
@@ -68,7 +59,7 @@ impl Config {
                     )))
                     .push_timeout(control.connect.timeout)
                     .push(control::client::layer())
-                    .push(discover)
+                    .push(discover::resolve(control::dns_resolve::Resolve::new(dns)))
```

Comment:
nit, take it or leave it: should there maybe be a `push_resolve` to shorten the several places this shows up? not a blocker.

Reply:
probably not. the layer is unnamable.

Reply:
oh, yeah, disregard this, nevermind.